contracts/src/v0.8/dev/keeper2_0/KeeperRegistrar2_0.sol

```diff
@@ -69,7 +70,7 @@ contract KeeperRegistrar2_0 is TypeAndVersionInterface, ConfirmedOwner, ERC677Re
     address adminAddress,
     bytes checkData,
     uint96 amount,
-    uint8 indexed source
```

Comment:
This is going to be removed anyway in https://github.com/smartcontractkit/chainlink/pull/7328  and I will rebase my changes on top of it.

Removing right now to prevent stack too deep